### PR TITLE
New package: bitwig-studio-3.2.6

### PR DESCRIPTION
**Pull request by ale-saglia**
My first time creating a package template! Hope to learn something.

Bitwig Studio is a multi-platform music-creation system for production, performance and DJing, with a focus on flexible editing tools and a super-fast workflow.

### CODE REVIEW

File: srcpkgs/bitwig-studio/template [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'bitwig-studio'
+
+pkgname=bitwig-studio
+homepage="bitwig.com"
+version=3.2.6
+revision=1
+archs="x86_64"
+short_desc="DAW for music production, remixing and live performance"
+
```
Comment (PaperMountainStudio) [inline srcpkgs/bitwig-studio/template:9]: Hi, could you remove these blank lines between variables? Then xlint will be able to tell you how to reorder lines.

File: srcpkgs/bitwig-studio/template [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+nostrip=yes
+
+depends="xdg-utils xcb-util-wm libbsd"
+hostmakedepends="tar xz"
+
+maintainer="Alessandro Saglia <ale.saglia@gmail.com>"
+license="custom:bitwig"
+
+_file="bitwig-studio-${version}.deb"
+_src="https://downloads.bitwig.com/stable/${version}"
```
Comment (PaperMountainStudio) [inline srcpkgs/bitwig-studio/template:21]: _src seems unneccessary here since it is used only once

**Comment by ale-saglia** [comment]
I think I've done something stupid and I delete my fork (bit Github noob here).
Do I have some way to recover and commit this pull request or do I need to refork and create another request)
Sorry again... 😓

**Comment by PaperMountainStudio** [comment]
I am sure there are a lot of ways, but you can do this in the repository:
```
wget https://github.com/void-linux/void-packages/pull/24095.patch
git am 24095.patch
```

**Comment by ale-saglia** [comment]
Ok I tried what you suggest me but it still asks to perform another pull request.
In the meantime I also fixed the problem in the formatting and unused variables.

**Comment by PaperMountainStudio** [comment]
It seems like the repository was deleted, so you really need to re-do the PR. Close this one and open a new one, probably.

**Comment by ale-saglia** [comment]
Ok thanks

**Pull request closed without merging**